dataset/home.htm

```diff
@@ -4,6 +4,7 @@ <h1>News</h1>
   <li>We are pleased to announce the COCO 2020 <a class="fontBoldish" href="#detection-2020">Detection</a>, <a class="fontBoldish" href="#keypoints-2020">Keypoint</a>, <a class="fontBoldish" href="#panoptic-2020">Panoptic</a>, and <a class="fontBoldish" href="#densepose-2020">DensePose</a> Challenges.<br/></li>
   <li class="fontBoldish">The new rules and awards for this year challenges encourage innovative methods.</li>
   <li>Results to be announced at the <a class="fontBoldish" href="workshop/coco-lvis-eccv-2020.html">Joint COCO and LVIS Recognition</a> ECCV workshop.</li>
+  <li>We have partnered with the team behind the open-source tool <a class="fontBoldish" href="https://fiftyone.ai">FiftyOne</a> to make it easier to download, visualize, and evaluate COCO</li>
```

Comment:
FiftyOne provides open-source tool to visualize and facilitate access to COCO data resources and an evaluation tool to breakdown the detection performance on COCO.